### DISCUSSION TITLE
connector-x on Anaconda?

### DISCUSSION THREAD

**Discussion by fearcon-eap** [sfu-db/connector-x#178] [Q&A]
Hey,

awesome work with connector-x.
I was wondering if you have thought about putting publishing it for (pure) conda users too?
In my team most people are using conda and while it works to pip install a package into a conda environment for dependency reasons it would be nice to have everything from the same source.

**Comment by wangxiaoying** [comment]
Hi @fearcon-eap , thanks for the suggestion. I believe publishing to conda would benefit many users. We will take a look into this.

**Reply by wseaton** [reply]
It looks like it should be possible to package connector-x similar to how maturin is being packaged by conda-forge: conda-forge/staged-recipes#10902

More research is required ofc.

**Reply by wangxiaoying** [reply]
Thanks @wseaton !
